api/spec/v1.yaml

```diff
@@ -2572,6 +2596,59 @@ components:
             the `/{runtime}/accounts/{address}` endpoint.
           example: false
 
+    EvmNftList:
+      allOf:
+        - $ref: '#/components/schemas/List'
+        - type: object
+          required: [evm_nfts]
+          properties:
+            evm_nfts:
+              type: array
+              items:
+                $ref: '#/components/schemas/EvmNft'
+              description: A list of L2 EVM NFT (ERC-721, ...) instances.
+          description: A list of NFT instances.
+
+    EvmNft:
```

Comment:
It feels awkward that we have some info here that's specific to an individual tokenID (like metadata_uri and image), and some that is a property of the token contract as a whole (most fields).

ERC721 are also already listed under `/{runtime}/evm_tokens`. How would it be if we expanded that response type to optionally include any contract-level fields that this PR newly introduces? Then this type could focus on just the per-ID properties (and maybe include the per-contract properties as a subtree, if we want to be verbose).

Reply:
this is for later when there's get NFT instances owned by an account

Reply:
Right. I didn't express myself clearly. I was talking about the _datatype_ of the `/{runtime}/evm_tokens` response, which is `#/components/schemas/EvmToken[List]`. Would it make sense to reuse `#/components/schemas/EvmToken` as a subtree of the new `EvmNft` type so that the other fields of `EvmNft` are mostly just things that are specific to that NFT ID?

Reply:
> Would it make sense to reuse `#/components/schemas/EvmToken` as a subtree of the new `EvmNft` type so that the other fields of `EvmNft` are mostly just things that are specific to that NFT ID?

I think that's a great idea.

> maybe include the per-contract properties as a subtree, if we want to be verbose

Yes I definitely think that we want to be verbose, otherwise we will potentially have to make multiple subsequent follow-up queries from the client side to get the missing data.

Reply:
oh is that structure bigger and does it contain other stuff you want for the UI?

Reply:
Not bigger, but different. In same cases, we need one, on some cases, the other, in same cases, both.

Reply:
moved "token" (terminology disputed) fields to a `token` subobject